versions.json

```diff
@@ -559,11 +491,16 @@
         "version_string": "0.36.0",
         "license": "Apache-2.0",
         "consortium_build_number": "3",
-        "package_revision": "0",
+        "package_revision": "2",
         "externals_root": "opt/irods-externals",
+        "patches": [
+            "qpid-proton/0001-PROTON-2450-Generate-correct-relocatable-pc-files-33.patch",
```

Comment:
Why is package revision not bumped to 1 instead of 2?

Reply:
It was bumped to 1 in one commit, and then to 2 in another.